modules/media/test/TestPlan.md

```diff
@@ -51,15 +51,13 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
     - A popup should prompt you to select a **Visit Label** as it is a required field ❌
   3. Select Visit Label and click on the 👉  **Upload file** again
     - A popup should prompt you to select a **Site** as it is a required field ❌
-  4. Select Site and click on the 👉  **Upload file** again
-    - A popup should prompt you to select a **File to upload** as it is a required field ❌
-  5. Click on the 👉 **Browse** button and a select a file from your file system
+  4. Click on the 👉 **Browse** button and a select a file from your file system
     - Note: it is suggested to try different file types and sizes (e.g pdf, mp4, mov, jpg, doc, etc)
-  6. After you selected the file click on **Upload File** one more time
+  5. After you selected the file click on **Upload File** one more time
```

Comment:
The "one more time" should be removed and I think the behaviour of "popup prompting to name the file" was removed.

Reply:
@maltheism I think the "popup prompting to name the file" refers to the case the file didn't  have been named properly before.

![image](https://user-images.githubusercontent.com/37309344/78917342-bfb34c00-7a5c-11ea-9be5-c1b45f0c89c0.png)

Reply:
regarding to the "one more time" statement I think is a question of style from the original document. I'm don't have any feeling against, neither in favour of removed it or leave it. In the case we take the first option, should we remove also the "again" statements in the following lines?   

2. Select PSCID and click on the 👉 Upload file button `again` - A popup should prompt you to select a Visit Label as it is a required field ❌
3. Select Visit Label and click on the 👉 Upload file `again` - A popup should prompt you to select a Site as it is a required field ❌

Reply:
@christinerogers would you happen to have an opinion on this?

Reply:
Whether we should remove the word `again/one more time` or not?  Sure, remove it.